scripts/build_provenance

```diff
@@ -10,7 +10,11 @@ readonly OUTPUT_PATH=$2
 # Shift arguments so that the remaining arguments are the just build arguments.
 shift 2
 
-./scripts/docker_run "${@}"
+# Run the build in a simplified docker configuration that uses the default
+# docker user and does not pass ENV vars. Flags are intentionally a subset of
+# the ones used in `./scripts/docker_run` which proxies host cache dirs and
+# file system permissions for developer convenience.
```

Comment:
Nice description, thanks for taking the time to write down the why (and not just the what).